CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Added support for Mean in DDP Sync ([#2568](https://github.com/PyTorchLightning/pytorch-lightning/pull/2568))
 
+- Added support for `limit_{mode}_batches (int)` to work with infinite dataloader(IterableDataset) ([2787](https://github.com/PyTorchLightning/pytorch-lightning/pull/2787))
```

Comment:
```suggestion
- Added support for `limit_{mode}_batches (int)` to work with infinite dataloader (IterableDataset) ([2787](https://github.com/PyTorchLightning/pytorch-lightning/pull/2787))
```

Reply:
Will make these changes :)

Reply:
you can just accept it here, so it may be easier them searching the lines in IDE, and also you can accept multiple suggestions as a single commit...